draft-illyes-cbcp.md

```diff
@@ -89,7 +101,24 @@ standard signals defined by
 [RFC9110](https://www.rfc-editor.org/rfc/rfc9110#name-server-error-5xx), preferably also
 additional heuristics such as relative response time of the server.
 
+As such, crawlers should log already visited URL and how many requests they sent to a resource 
+as well as the respective HTTP status codes in the responses,
+especially if error occur, to avoid repeatedly crawling the same source.
+
+Generally, crawlers should avoid multiple concurrent requests to the same ressources
+and use a crawling rate that is no higher than 1 request per second or as specified in
+the REP protocol. Further, ressources should not be re-crawled more often than every 24 hours.
+Ideally crawlers should limit the crawling depth and number of requests
+per ressource to avoid loops.
+
+Crwalers should not attempt to access restricted ressources that requires a login, used CAPTCHAs, or are behind paywalls.
+
+Crawlers should only access content and by all means avoid to modify content by using
+HTTP request other than GET such as POST, PUT, DELETE. Similarly crawlers should avoid
+assessing Javascript content. 
```

Comment:
JS content is imperative for a decent index nowadays. This is not going to work.
https://developers.google.com/search/docs/crawling-indexing/javascript/javascript-seo-basics

Reply:
Yes, I thought that was to restrictive but I thought I wait what you say... Are you crawling all JS content or do you select somehow?